Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -24,19 +24,14 @@ import PayPalCheckout
     /// `BTPayPalNativeCheckoutAccountNonce`. On failure or user cancelation you will receive an error. If the user cancels
     /// out of the flow, the error code will equal `BTPayPalNativeError.canceled.rawValue`.
     /// - Parameters:
-    ///   - nativeRequest: Either a BTPayPalNativeCheckoutRequest or a BTPayPalNativeVaultRequest
+    ///   - request: Either a BTPayPalNativeCheckoutRequest or a BTPayPalNativeVaultRequest
     ///   - completion: The completion will be invoked exactly once: when tokenization is complete or an error occurs.
     @objc(tokenizePayPalAccountWithPayPalRequest:completion:)
     public func tokenizePayPalAccount(
-        with nativeRequest: BTPayPalRequest,
+        with request: BTPayPalNativeRequest,
         completion: @escaping (BTPayPalNativeCheckoutAccountNonce?, Error?) -> Void
     ) {
         apiClient.sendAnalyticsEvent("ios.paypal-native.tokenize.started")
-        guard let request = nativeRequest as? (BTPayPalRequest & BTPayPalNativeRequest) else {
-            apiClient.sendAnalyticsEvent("ios.paypal-native.tokenize.invalid-request.failed")
-            completion(nil, BTPayPalNativeError.invalidRequest)
-            return
-        }
```

Comment:
Since the `request` is always of type `BTPayPalNativeRequest` now we no longer need this guard as it would emit a warning stating that request is always of type `BTPayPalNativeRequest`